src/sentry/models/projectownership.py

```diff
@@ -12,6 +12,7 @@
 from sentry.utils import metrics
 from sentry.utils.cache import cache
 from functools import reduce
+from .projectcodeowners import ProjectCodeOwners
```

Comment:
In the models directory we just import other models inside the functions that use them.